src/services/review/ReviewRequestService.ts

```diff
@@ -299,6 +299,35 @@ export default class ReviewRequestService {
     )
   }
 
+  markReviewRequestAsViewed = async (
+    sessionData: UserWithSiteSessionData,
+    site: Site,
+    requestId: number
+  ): Promise<void> => {
+    const { isomerUserId: userId } = sessionData
+
+    const reviewRequestView = await this.reviewRequestView.findOne({
```

Comment:
nit: could we try doing an `upsert` instead? see [here](https://sequelize.org/docs/v6/other-topics/upgrade/#modelupsert) for more details.

Reply:
Hmm I think it might not be possible, because we only want to insert if it is not found. We do not want to perform any updates to the existing entry.

However, I think using `upsert` would be very useful for `updateReviewRequestLastViewedAt`, updated that in 927116d.